packages/device_info/ios/Tests/unit_test.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Created by TruongSinh Tran-Nguyen on 2019-01-05.
+//  Copyright © 2019 The Chromium Authors. All rights reserved.
```

Comment:
I think this should be changed to the standard Flutter copyright block

Reply:
Thanks, it's done in the latest commit